### PR TITLE
Update README - Alternatives

### PR DESCRIPTION
**Pull request by IPv2**
* Add link to mu/mu4e, another project which [courteously back-links to Sup](https://www.djcbsoftware.nl/code/mu/mu4e/Other-mail-clients.html)
* Add link to Alot screenshots, showcasing "Sup" style theme
* Typo fix, s/frontend/front end/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,9 +57,15 @@ some alternatives to consider:
 * [Notmuch](https://notmuchmail.org/) was inspired by Sup. There are a wide
   variety of [Notmuch clients](https://notmuchmail.org/frontends/) available.
   The most similar to Sup's look-and-feel is
-  [Alot](https://github.com/pazz/alot) &mdash; also a curses-based frontend.
-  Alot even ships with a built-in
-  [Sup theme](https://github.com/pazz/alot/blob/master/extra/themes/sup)!
+  [Alot](https://github.com/pazz/alot) &mdash; also a curses-based front end.
+  Alot even ships with a
+  [built-in](https://github.com/pazz/alot/blob/master/extra/themes/sup)
+  [Sup theme](https://github.com/pazz/alot/wiki/Gallery#user-content-theme-sup)!
```
Comment (danc86) [inline README.md:63]: It looks like the correct anchor link is:
https://github.com/pazz/alot/wiki/Gallery#theme-sup

**Comment by danc86** [comment]
This looks good to me, but I would like to hold off on merging anything right now.

I am giving some thought to re-releasing 0.23 with a version number of 1.0. It feels wrong to me that sup is still calling itself 0.x after all these years, when it has been quite stable. Maybe it's finally time to call 1.0?

I'm thinking about it now because it would make things simpler for everyone if I can say that 1.0 is identical to 0.23 aside from the version number.

File: README.md (hunk 1 above)
Comment (IPv2) [reply README.md:63]: GitHub has relatively recently started to add the prefix `user-content-` - here's how it appears in the rendered HTML: `<a id="user-content-theme-sup" class="anchor" href="#theme-sup" aria-hidden="true">`

I found that anchor `#theme-sup` therefore didn't work (despite what the browser URL bar told me), and had to use `#user-content-theme-sup` instead.

This is similar to, e.g., https://github.com/MicrosoftDocs/azure-devops-docs/issues/6015 and https://github.com/revolunet/sublimetext-markdown-preview/issues/208#issuecomment-202945428

**Comment by IPv2** [comment]
Firstly a big thank you for releasing 0.23.

I agree - after 14 years of development, and 5+ years of no breaking changes, we'd be well within the spirit of SemVer to call 1.0. (Overdue, if anything.)

Perhaps non-core code (documentation, possibly tests) could be excluded from this, i.e., you could then say that 1.0 is code-identical to 0.23, and this would keep things simple enough? (It would also be nice to fix our skipped test before 1.0, for example!)

**Comment by danc86** [comment]
Well there are always more things we could fix :-) but 1.0 doesn't have to mean perfect -- just that we're not intending on breaking compatibility for users. Since Sup is a user-facing program instead of a library, the interfaces that matter are things like default keybindings, hook interfaces, config files... and those have been stable for a long time and I don't intend on breaking compatibility there.

So I think I have convinced myself to just re-release it as 1.0. I'm going to do that now.

File: README.md (hunk 1 above)
Comment (danc86) [reply README.md:63]: Hmm yeah it's a bit strange. I wonder if they are using some Javascript to mess with the anchors when the page loads. If you scroll down to the "Theme sup" heading and click the link icon that appears when you hover, it goes to:

https://github.com/pazz/alot/wiki/Gallery#theme-sup

which correctly jumps back to that heading.

But when I try navigating directly to that URL, or to:

https://github.com/pazz/alot/wiki/Gallery#user-content-theme-sup

my browser just stays at the top of the page. So I'm not sure what is going on. Doesn't really matter I guess.

**Comment by IPv2** [comment]
Releases are like buses - you wait five years, and then two come at once. :-) Thank you for releasing 1.0!

**Pull request merged**